NERODevelopment/src/models/raspberry_model.cpp

```diff
@@ -338,6 +338,19 @@ std::optional<int> RaspberryModel::getNumberOfNonCriticalFaults() {
   return std::nullopt;
 }
 
+std::optional<float> RaspberryModel::getFastestTime() {
```

Comment:
yeah these wont be got by id, they need to be stored inside the model and then when we check for the current time, if its faster than the fastest time were gonna set it to that. but this should get implemented in the model, not the raspberry model